packages/react-components/review/beta/react-components.api.md

```diff
@@ -490,11 +490,11 @@ export interface ChatMessage extends MessageCommon {
 // @public
 export interface ChatTheme {
     chatPalette: {
-        modalOverlayBlack: string;
-        modalTitleWhite: string;
-        modalButtonBackground: string;
-        modalButtonBackgroundHover: string;
-        modalButtonBackgroundActive: string;
+        imageGalleryOverlayBlack: string;
+        imageGalleryTitleWhite: string;
+        imageGalleryDefaultButtonBackground: string;
+        imageGalleryButtonBackgroundHover: string;
```

Comment:
```suggestion
        imageGalleryDefaultButtonBackgroundHover: string;
```

(Unless all buttons and future buttons (dismiss included) use the same background hover color)

Reply:
Currently the hover and active color are used by both the default download button and the close button